monitoring/configuration/prometheus-rbac.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: prometheus
```

Comment:
Do we need to define this? Isn't it part of the monitoring infrastructure that is already present?

Reply:
Fixed - moved this file to the prometheus directory instead.